app/src/main/java/com/sopt/dive/ui/screens/ProfileScreen.kt

```diff
@@ -0,0 +1,46 @@
+package com.sopt.dive.ui.screens
+
+import android.content.Intent
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.sopt.dive.model.User
+import com.sopt.dive.ui.LoginActivity
+
+@Composable
+fun ProfileScreen(paddingValues: PaddingValues, user: User) {
+    val context = LocalContext.current
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(paddingValues)
+            .padding(20.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.Center
+    ) {
+        Text(" Profile", fontSize = 28.sp, fontWeight = FontWeight.Bold, textAlign = TextAlign.Center)
```

Comment:
컴포저블에서 여러 개의 파라미터 속성을 사용할 경우, 각 파라미터의 이름을 명시해주는게 좋습니다!
저번 세미나 때도 말씀 드렸는데 파라미터명을 생략하면 여러 속성을 사용할 때 잘못 인식될 수도 있기 때문입니다!

여기랑 다른 곳에서 몇 개는 파라미터명을 쓰시고 몇 개는 안 쓰신 경우가 있던데 일관적으로 다 적어주시면 좋을 것 같습니다!

```suggestion
        Text(text = " Profile", fontSize = 28.sp, fontWeight = FontWeight.Bold, textAlign = TextAlign.Center)
```

Reply:
아 감사합니다 아직 미흡한 부분이 많네요 관련해서 리팩토링 진행해보겠습니다 !